### ISSUE TITLE
Is there a reason for the large difference in training and validation loss with pt-forecasting models?

### ISSUE THREAD

**Issue by chododom** [jdb78/pytorch-forecasting#1454]
I have experimented with several models from pytorch-forecasting (TFT, DeepAR, LSTM) and I have noticed, that in all the cases, the training loss is very different to validation loss and it does not seem to be an issue concerning model performance (overfitting/underfitting) or weird data separation.
Is it possible, that the loss is calculated on transformed features for one prediction and not for the other?
